cmd/cmd.go

```diff
@@ -29,6 +32,17 @@ func Run() error {
 		fmt.Printf("hi: %v\n", *quiet)
 	}
 
+	// getting password if it isn't passed in
+	if len(*password) == 0 {
+		fmt.Print("Enter Password: ")
+		bytepw, err := term.ReadPassword(int(syscall.Stdin))
```

Comment:
```suggestion
		bytepw, err := term.ReadPassword(os.Stdin)
```
youd also need to fix the imports

Reply:
ez fix

Reply:
ReadPassword() expects an int fd, so the change isn't exactly what you put but I think this should be sufficient